system_tests/lewis_emulators/stanford_sr400_photon_counter/device.py

```diff
@@ -34,18 +34,18 @@ def set_random_status(self) -> None:
 
     def get_count_a(self) -> int:
         if self.counting:
-            self.count_a += random.randint(10, 15)
+            self.count_a = 20
         return self.count_a
 
     def get_count_b(self) -> int:
         if self.counting:
-            self.count_b += random.randint(10, 15)
+            self.count_b = 15
         return self.count_b
     
     def get_status(self) -> int:
-        if self.counting:
+        #if self.counting:
             # change status on
-            self.set_random_status()
```

Comment:
- Can we just remove this code instead of commenting it out?
- Do we still need the `set_random_status` function or can it be removed?
- Once the above 2 points are done can we remove `import random` from the top of the file.

Reply:
This change has been done